xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/usersandgroups/usersandgroups.js

```diff
@@ -55,34 +64,62 @@ window.MSCheckbox = Class.create({
     ];
     this.labels = ['','',''];
 
+    
+    
+    this.button = document.createElement("button");
+    this.button.className = "rights-edit";
+    Event.observe(this.button, 'click', this.createClickHandler(this));
```

Comment:
Is there any reason for using Prototype here? We're moving away from Prototype so I don't think it is a good idea to expand it's usage unless it is strictly required (see also, e.g., https://jira.xwiki.org/browse/XWIKI-17645).

Reply:
Thanks! I made the changes too quickly, I reverted it back to the way they added the listener before this PR, and didn't realize it was still different from the way you linked [above](https://github.com/xwiki/xwiki-platform/pull/2051#discussion_r1088768833).